comp110_architecture.tex

```diff
@@ -0,0 +1,80 @@
+% Please do not change the document class
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+
+% Please include a clear, concise, and descriptive title
+\title{An Analysis of the Variability, Reliability, and Control Provided by Level Generation Techniques for 2D Platform Games}
+
+% Please do not change the subtitle
+\subtitle{COMP110 - Computer Architecture Essay}
+
+% Please put your student number in the author field
+\author{1507290}
+
+\begin{document}
+
+\maketitle
+
+\abstract{There are a number of diverse approaches to level generation for 2D platform games, each placing emphasis on a different aspect of level generation. Chunk-based approaches such as Occupancy Regulated Extension tend to favour control and variation, whilst rhythm-based approaches such as that demonstrated by Launchpad favour playability and enjoyability. There are also evolutionary approaches that use genetic algorithms, which when combined with constraint satisfaction, as demonstrated by Sorenson et al., provide a high level of control and reliability.}
+
+\section{Introduction}
+Procedural content generation for 2D platform games is a relatively new field in comparison to games of other genres~\cite{compton:platform}. Many diverse approaches to this problem have been developed, each having advantages and disadvantages.~\cite{horn:comparative}. There are chunk-based approaches which rely on hand-designed chunks of level; rhythm-based models that generate a level based on sequences of player actions; and evolutionary approaches that use genetic algorithms. It could be argued that the most desirable level generator would be the one that requires the least human input yet offers high amounts of control and is able to generate a wide variety of fun and playable levels consistently. 
+
+\section{Comparison}
+
+\subsection{Overview}
```

Comment:
The overview is clear an format is excellent because it emphasizes analysis over description. I would suggest introducing figures to illustrate the algorithms and how they work as this will improve the context of your comparisons for readers who are unfamiliar with the works you are citing.